app/controllers/api/v1/projects_controller.rb

```diff
@@ -128,14 +134,15 @@ def build_resource_for_create(create_params)
 
     content_attributes = primary_content_attributes(create_params)
     create_params[:project_contents] = [ ProjectContent.new(content_attributes) ]
+    create_params[:url_labels] = content_attributes[:url_labels]
```

Comment:
It may not do but does this need the associated `urls` attribute as well?

Reply:
No, that's updated in place in the original params hash (which is horrible but out of scope unless you want me to clean it up)